iam-persistence/src/main/resources/db/migration/h2/V106___add_reader_authority.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO iam_authority(AUTH) VALUES
```

Comment:
Update it to V107 or more because we recently added a PR with a migration number equal to V106

Reply:
Done